### ISSUE TITLE
Set dotStyle individually [v2]

### ISSUE THREAD

**Issue by AntseN** [NightCatSama/vue-slider-component#362]
#### Describe the bug

I use to be able to style the dots individually by doing the following, in a previous version of your component :  
```js
 sliderStyle: [
                        {
                            "backgroundColor": "#ffffff",
                            "border": "1px solid #e0e0e0",
                            "boxShadow": "0.5px 0.5px 2px 1px rgba(107,107,107,.36)"
                        },
                        {
                            "backgroundColor": "#efb900",
                            "border": "1px solid #e0e0e0",
                            "boxShadow": "0.5px 0.5px 2px 1px rgba(107,107,107,.36)"
                        }
                    ],
```

In the latest version (as of today) it does not work, and the documentation is not clear about how to pass styles anymore. I only figured out how to style both dots at the same time like so :   
```js
dotStyle:
                        {
                            "backgroundColor": "#efb900",
                            "border": "1px solid #e0e0e0",
                            "boxShadow": "0.5px 0.5px 2px 1px rgba(107,107,107,.36)"
                        }
```

What am i missing ?

#### Environment 

- last version of vue & your component.

**Comment by NightCatSama** [comment]
Can be set with `dotOptions`. [example](https://jsfiddle.net/u5rmckfx/)

document: <https://nightcatsama.github.io/vue-slider-component/#/api/props?hash=dotoptions>

**Comment by AntseN** [comment]
Indeed thx for the fast reply, i couldnt understand dotOptions usage.

**Comment by nithinkjoy-tech** [comment]
> https://nightcatsama.github.io/vue-slider-component/#/api/props?hash=dotoptions

Hi @NightCatSama, Is this `dotOptions` available in `v2.8.16`?
